docs/1.3_to_1.4_migration_guide.md

```diff
@@ -1,15 +1,25 @@
-# List of broken changes for migration from indy-node 1.3 to 1.4:
+# List of broken changes for migration from indy-node 1.3 to 1.4
 
 -**Another format of REPLY for "WRITE" transactions**. New REPLY format you can see in:
  [REPLY for "WRITE" txns](https://github.com/hyperledger/indy-node/blob/master/docs/requests.md#reply-structure-for-write-requests).
  The main goal for this changes is to divide data and metadata.
 
--**Change key of ReqIdrToTxn store**.   For now, ReqIdrToTxn store request from client as a map of:
+-**Change key of ReqIdrToTxn stores**.   For now, ReqIdrToTxn store request from client as a map of:
  (digest) -> (ledger_id<some_delimiter>txn seq_no). Digest is a sha256 hash of (reqId, DID, operation). 
  Also, digest was moved to transaction metadata section.
  
--**Added tag for CLAIM_DEF**. 'TAG' is a part of key for state tree. If this field exist in incoming transaction, then it will be used. If not, then default.  
+-**Added tag for CLAIM_DEF**. 'TAG' is a part of key for state tree. If this field exist in incoming transaction, then it will be used. If not, then default will be used.  
 Full CLAIM_DEF format description you can see in: [CLAIM_DEF](https://github.com/hyperledger/indy-node/blob/master/docs/transactions.md#claim_def)
```

Comment:
`Full CLAIM_DEF format description can be found in [CLAIM_DEF txn format](https://github.com/hyperledger/indy-node/blob/master/docs/transactions.md#claim_def) and [CLAIM_DEF request format](https://github.com/hyperledger/indy-node/blob/master/docs/requests.md#claim_def)`